packages/circuit-ui/components/SelectorGroup/SelectorGroup.spec.tsx

```diff
@@ -81,7 +78,12 @@ describe('SelectorGroup', () => {
   });
 
   it('should render with horizontal layout by default', () => {
-    const actual = create(<SelectorGroup {...defaultProps} />);
+    const actual = render(<SelectorGroup {...defaultProps} />);
+    expect(actual).toMatchSnapshot();
+  });
+
+  it('should render all selectors with the same height', () => {
+    const actual = render(<MultipleLines />);
     expect(actual).toMatchSnapshot();
   });
```

Comment:
I'm questioning how much value this additional test adds. The snapshot is almost identical to the one from the previous test, apart from the option with a long label. The CSS that ensures that long labels are correctly rendered is included in both tests.

I think the Chromatic snapshot for the story you added is enough to prevent regression.

Reply:
I agree, I wanted to create a test case regarding components with different heights, but this test really doesn't add much. I've removed it.

Reply:
`I think the Chromatic snapshot for the story you added is enough to prevent regression.`
Do I need to do something in order to trigger a `Chromatic Snapshot` besides writing the story? I don't have experience with Chromatic

Reply:
Nevermind, already found my way with Chromatic

Reply:
Nevermind this, I've already found my way with Chromatic

Reply:
For anyone reading this in the future: Chromatic is run as a continuous integration (CI) check. It automatically generates snapshots of each story and compares them to their previous version. Any changes need to be approved in their web app which can be accessed by anyone who's a member of the SumUp OSS org on GitHub.